lib/utils.pm

```diff
@@ -393,9 +393,8 @@ that.
 sub check_console_font {
     # Does not make sense on ssh-based consoles
     return if get_var('BACKEND', '') =~ /ipmi|spvm|pvm_hmc/;
-    # we do not await the console here, as we have to expect the font to be broken
-    # for the needle to match
```

Comment:
Doesn't this apply anymore?

Reply:
Need remove wait_console. So it can wait console login to root then put the font command.

Reply:
But this will now make the `select_console` call check for a console prompt needle before the consolefont is checked.

Reply:
@foursixnine WDYT?

Reply:
Our problem is, before login root console, it will input font command, it will make consoletest_setup failed for example: https://openqa.suse.de/tests/5217977#step/consoletest_setup/56

Reply:
I don't understand the comment, when there is no console then there is no output e.g. the failure (I guess this was there before virtio_console was used).
There are so many failures are you sure this will solve the problem ?
Looks like problem on slow aarch64 only, change it for this arch ?

Reply:
This can solve the failures, after root console show up, then input the font command. I will change code only for our migration test. As for migration other arch like ppc64le also has this problem.